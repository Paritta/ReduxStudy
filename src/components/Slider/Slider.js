import React from "react";

const propTypes = {
};

const defaultTypes = {
};

export const Slider = () => {
    return (
        <div>
            Slider
        </div>
    )
};

Slider.propTypes = propTypes;
Slider.defaultTypes = defaultTypes;

export default Slider;
