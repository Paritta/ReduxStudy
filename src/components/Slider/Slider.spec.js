import React from "react";
import { Slider } from "./Slider";
import "jest-styled-components";
import ReactTestRenderer from "react-test-renderer";

it("Card render correctly", () => {
    const tree = ReactTestRenderer.create(
        <Slider />
    );
    expect(tree.toJSON()).toMatchSnapshot();
});
