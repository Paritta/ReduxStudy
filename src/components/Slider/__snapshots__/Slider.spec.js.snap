// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
<div>
  Slider
</div>
`;
